platforms/quorum/charts/certs-ambassador-quorum/Chart.yaml

```diff
@@ -8,4 +8,4 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm3 chart for Ambassador certificates creation, the purpose of this Helm Chart is to generate SSL/TLS certificates using OpenSSL, including a root CA certificate and node certificates, and storing them in a Vault server. These certificates enable secure communication and authentication between servers and clients in a system.
 name: certs-ambassador-quorum
-version: '0.14.0'
+version: '0.14.1'
```

Comment:
All the charts need not have a version update, only the ones that you have changed.